realgud/common/track.el

```diff
@@ -168,6 +168,23 @@ message."
              "Buffer %s is not a debugger command buffer" buf)
     t))
 
+(defun realgud:get-output-command(text)
+  "Splits the TEXT by newline."
+  (car (split-string text "\n")))
+
+(defun realgud:get-eval-output(text)
+  "Gets the output stripping the command and debugger prompt from the TEXT."
+  (string-join (butlast (cdr (split-string text "\n"))) "\n"))
+
```

Comment:
string-join is emacs 25 only, not sure if thats ok.

Reply:
That could be problematic. If this is to run emacs 25 only, then we'd need to update lines like https://github.com/realgud/realgud/blob/master/realgud.el#L79

Is there a comparability package for emacs 24? Or if not maybe we could cut and paste that in guarded by a test on the emacs version?

Reply:
oh i'm sure this will be simple to do without string-join, just a little less pretty

Reply:
How about if you code string-join which gets defined in  `(if (< emacs-major-version 25)...` and put in https://github.com/realgud/realgud/blob/master/realgud/common/utils.el ?

Reply:
There are other string-like things in utils.el. If you see some that now have Emacs 25 counterparts, let me know and I'll change the names and do likewise.